ros_buildfarm/templates/doc/doc_independent_site_job.xml.em

```diff
@@ -0,0 +1,135 @@
+<project>
+  <actions/>
+  <description>Generated at @ESCAPE(now_str) from template '@ESCAPE(template_name)'</description>
+  <keepDependencies>false</keepDependencies>
+  <properties>
+@(SNIPPET(
+    'property_log-rotator',
+    days_to_keep=365,
+    num_to_keep=100,
+))@
+@[if job_priority is not None]@
+@(SNIPPET(
+    'property_job-priority',
+    priority=job_priority,
+))@
+@[end if]@
+@(SNIPPET(
+    'property_requeue-job',
+))@
+  </properties>
+@(SNIPPET(
+  'scm_git',
+   url=upload_repository_url,
+   branch_name=upload_repository_branch or 'gh-pages',
```

Comment:
Is there a reason to prefer setting a default in the template rather than setting it earlier on or not setting a default and asserting?

Reply:
It is actually being set before. A leftover, I'll remove it.